src/assets/sass/protocol/templates/_card-layout.scss

```diff
@@ -0,0 +1,204 @@
+// Card grid layout classes.
+
+/* -------------------------------------------------------------------------- */
+// 5 card hero
+// On small screens cards stack vertically and span full container width.
+// On medium sized screens the 1st card spans full width, and the remaining 4 cards span half widths.
+// On large screens the 1st card spans 2 thirds, and the remaining cards span 1 third.
+.mzp-l-card-hero {
+    @media #{$media-sm} {
+        @include clearfix;
+
+        .mzp-c-card {
+            @include bidi((
+                (float, left, right),
+                (margin-left, 0, $margin-xl),
+                (margin-right, $margin-xl, 0),
+            ));
+            width: calc(50% - (#{$margin-xl} / 2));
+
+            &:nth-child(odd) {
+                margin-left: 0;
+                margin-right: 0;
+            }
+
+            &.mzp-c-card-large {
+                width: 100%;
+            }
+        }
+    }
+
+    @media #{$media-lg} {
+        .mzp-c-card {
+            width: calc(33.3% - (#{$margin-xl} - (#{$margin-xl} / 3)));
+
+            &:nth-child(odd) {
+                @include bidi((
+                    (margin-left, 0, $margin-xl),
+                    (margin-right, $margin-xl, 0),
+                ));
+            }
+
+            &:nth-child(2),
+            &:last-child {
+                margin-right: 0;
+
+                [dir="rtl"] & {
+                    margin-left: 0;
+                }
+            }
+
+            &:nth-child(3n) {
+                @include bidi(((clear, left, right),));
+            }
+        }
+
+        .mzp-c-card.mzp-c-card-large {
+            width: calc(66.6% - (#{$margin-xl} / 3));
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// 4 card layout
+// On small screens cards stack vertically and span full container width.
+// On medium sized screens cards span half width.
+// On large screens cards span quater widths.
+.mzp-l-card-quater {
+    @media #{$media-sm} {
+        @include clearfix;
+
+        .mzp-c-card {
+            @include bidi((
+                (float, left, right),
+                (margin-left, 0, $margin-xl),
+                (margin-right, $margin-xl, 0),
+            ));
+            width: calc(50% - (#{$margin-xl} / 2));
+
+            &:nth-child(even) {
+                margin-left: 0;
+                margin-right: 0;
+            }
+        }
+    }
+
+    @media #{$media-lg} {
+        .mzp-c-card {
+            @include bidi((
+                (margin-left, 0, $margin-xl),
+                (margin-right, $margin-xl, 0),
+            ));
+            width: calc(25% - (#{$margin-xl} - (#{$margin-xl} / 4)));
+
+            &:nth-child(even) {
+                @include bidi((
+                    (margin-left, 0, $margin-xl),
+                    (margin-right, $margin-xl, 0),
+                ));
+            }
+
+            &:nth-child(4n) {
+                margin-right: 0;
```

Comment:
Another place where I think the mixin should work with four values: `@include bidi(((margin-right, 0, margin-left, 0),));`